examples/templating/src/tests.rs

```diff
@@ -40,17 +40,20 @@ fn test_404(base: &str) {
     let client = Client::tracked(rocket()).unwrap();
     for bad_path in &["/hello", "/foo/bar", "/404"] {
         let path = format!("/{}{}", base, bad_path);
-        let escaped_path = RawStr::new(&path).html_escape();
+
+        let escaped_path_upper = RawStr::new(&path).html_escape();
+        let escaped_path_lower = escaped_path_upper.to_lowercase();
```

Comment:
I've ran this modified test with command `cargo test -- --nocapture`:

```rust
fn test_404(base: &str) {
    // Check that the error catcher works.
    let client = Client::tracked(rocket()).unwrap();
    for bad_path in &["/hello", "/foo/bar", "/404"] {
        let path = format!("/{}{}", base, bad_path);
        let escaped_path = RawStr::new(&path).html_escape();

        let response = client.get(&path).dispatch();
        assert_eq!(response.status(), Status::NotFound);
        let response = response.into_string().unwrap();
        
        println!("path: '{}'", path);
        println!("escaped_path: '{}'", escaped_path);
        println!("response body: {:?}", response);
    }
}
```

And the case is different for the HTML escaped characters in escaped_path and response.

For `tera`:
```
path: '/tera/foo/bar'
escaped_path: '&#x2F;tera&#x2F;foo&#x2F;bar'
response body: "... The page at &#x2F;tera&#x2F;foo&#x2F;bar does not exist! ..."
```

For `minijinja`:
```
path: '/minijinja/foo/bar'
escaped_path: '&#x2F;minijinja&#x2F;foo&#x2F;bar'
response body: "... The page at &#x2f;minijinja&#x2f;foo&#x2f;bar does not exist! ..."
```

Is this good enough or is there a better solution?

Reply:
Fixed, please have a look!